codersdk/client.go

```diff
@@ -79,6 +79,10 @@ type Client struct {
 	HTTPClient *http.Client
 	URL        *url.URL
 
+	// TokenHeader is an optional custom header to use for setting tokens. By
+	// default SessionTokenHeader is used.
+	TokenHeader string
```

Comment:
This should be called `SessionTokenHeader` instead to align with `SetSessionToken`.